addons/event/models/event.py

```diff
@@ -216,6 +232,16 @@ def _compute_seats(self):
                 event.seats_available = event.seats_max - (event.seats_reserved + event.seats_used)
             event.seats_expected = event.seats_unconfirmed + event.seats_reserved + event.seats_used
 
+    @api.depends('stage_id', 'kanban_state')
+    def _compute_kanban_state_label(self):
```

Comment:
@tde-banana-odoo see above, I cannot remove those field, otherwise I have the wrong "classic" labels in the form view

Reply:
What I said is to use the one defined on the stage in the compute (label = stage.label, not the related ont). Otherwise you are missing dependencies and you cannot depends on unstored fields.